smoke-test/src/test/java/org/opennms/smoketest/minion/MinionHeartbeatOutageKafkaIT.java

```diff
@@ -38,8 +37,8 @@
  * 
  * @author Seth
  */
-@Category(MinionTests.class)
-//@org.junit.experimental.categories.Category(org.opennms.smoketest.junit.FlakyTests.class)
+//@Category(MinionTests.class)
+@Category(org.opennms.smoketest.junit.FlakyTests.class)
```

Comment:
...again? 🙃

Reply:
I started looking into it, and in the process of researching it, I noticed that the similar test that this one is extended from, `MinionHeartbeatOutageIT`, was marked as a flaky test in https://github.com/OpenNMS/opennms/commit/c1a1ca59f32a025600f473f5078d074e2de948b7

So I did the same thing here, since it looks like there's something common between the two that makes them flaky.

Reply:
yeah they keep getting flaked and unflaked, they obviously need to be rewritten to not be flaky